code/game/objects/items/devices/flashlight.dm

```diff
@@ -45,6 +119,23 @@
 		A.UpdateButtonIcon()
 	return 1
 
+/obj/item/flashlight/attackby(obj/item/I, mob/user)
+	if(istype(I, /obj/item/stock_parts/cell))
+		if(uses_battery)
+			if(int_battery)
+				to_chat(user, "<span class='warning'>There's already a cell inside!</span>")
+				return
+			int_battery = I
+			I.forceMove(src)
+			to_chat(user, "<span class='notice'>You plug the cell into \the [src].")
```

Comment:
No `</span>`

Reply:
Interpretation: no </span> means you forgot to put.